ci/ci.yaml

```diff
@@ -0,0 +1,217 @@
+          - task: jest
+            image: node
+            config:
+              platform: linux
+              inputs:
+                - name: pull-request
+                  path: .
+              caches:
+                - path: node_modules
+                - path: .yarn
+              run:
+                path: /bin/sh
+                args:
+                  - -exc
+                  - |
+                    yarn install --cache-folder .yarn
+                    yarn run jest
+          - task: eslint
+            image: node
+            config:
+              platform: linux
+              inputs:
+                - name: pull-request
+                  path: .
+              caches:
+                - path: node_modules
+                - path: .yarn
+              run:
+                path: /bin/sh
+                args:
+                  - -exc
+                  - |
+                    yarn install --cache-folder .yarn
+                    yarn run eslint src
+          - task: yamllint
+            image: python
+            config:
+              platform: linux
+              inputs:
+                - name: pull-request
+                  path: .
+              caches:
+                - path: cache
+              run:
+                path: /bin/sh
+                args:
+                  - -exc
+                  - |
+                    pip install --cache-dir ./cache yamllint
+                    yamllint .
+        on_failure:
+          put: pull-request
+          params:
+            path: pull-request
+            status: failure
+      - put: pull-request
+        params:
+          path: pull-request
+          status: success
+  - name: set-pipeline
+    plan:
+      - get: senior-matchmaking-main
+        trigger: true
+      - set_pipeline: self
+        file: senior-matchmaking-main/ci/ci.yaml
+  - name: publish
+    plan:
+      - get: senior-matchmaking-main
+        trigger: true
+      - get: oci-build-task
+      - get: sentry-cli
+      - task: sentry-new
+        image: sentry-cli
+        params:
+          SENTRY_ORG: ((sentry.org))
+          SENTRY_AUTH_TOKEN: ((sentry.auth-token))
+          SENTRY_URL: ((sentry.url))
+        config:
+          platform: linux
+          inputs:
+            - name: senior-matchmaking-main
+              path: .
+          outputs:
+            - name: release
+          run:
+            path: /bin/sh
+            args:
+              - -exc
+              - |
+                VERSION=senior-matchmaking@$(sentry-cli releases propose-version)
+                echo $VERSION > release/version.txt
+                sentry-cli releases new -p senior-matchmaking "$VERSION"
+                sentry-cli releases set-commits --auto "$VERSION"
+      - load_var: sentry-release
+        file: release/version.txt
+      - task: build
+        privileged: true
+        image: oci-build-task
+        params:
+          BUILD_ARG_RELEASE: ((.:sentry-release))
```

Comment:
Unused in Dockerfile?

Reply:
Fixed in 1dabfd76be8637e7a32c37bdd56c731038f0f0f3